tests/acquisition/covidcast/test_covidcast_row.py

```diff
@@ -22,9 +22,9 @@ class TestCovidcastRows(unittest.TestCase):
         "source": ["src"] * 10,
         "signal": ["sig_base"] * 5 + ["sig_other"] * 5,
         "time_type": ["day"] * 10,
-        "geo_type": ["county"] * 10,
+        "geo_type": ["msa"] * 10,
         "time_value": [2021_05_01 + i for i in range(5)] * 2,
-        "geo_value": ["01234"] * 10,
+        "geo_value": ["40660"] * 10,
```

Comment:
question: should this test use a value from test_utils.MSA instead?

Reply:
yes